src/main/scala/io/scala/Page.scala

```diff
@@ -100,20 +100,20 @@ object Page {
     decode = IndexPage(_, _),
     (root / endOfSegments) ? draftParam & conferenceParam
   )
-  val talksRoute = Route.onlyQuery[TalksPage, (Option[Boolean], Option[String])](
+  val sessionsRoute = Route.onlyQuery[SessionsPage, (Option[Boolean], Option[String])](
     encode = x => (x.withDraft, x.conference),
-    decode = TalksPage(_, _),
-    (root / "talks" / endOfSegments) ? draftParam & conferenceParam
+    decode = SessionsPage(_, _),
+    (root / "sessions" / endOfSegments) ? draftParam & conferenceParam
   )
-  val talkRoute = Route[TalkPage, (String, String)](
+  val sessionRoute = Route[SessionPage, (String, String)](
     encode = x => (x.conference, x.slug),
-    decode = TalkPage(_, _),
-    (root / "talks" / segment[String] / segment[String] / endOfSegments)
+    decode = SessionPage(_, _),
+    (root / "sessions" / segment[String] / segment[String] / endOfSegments)
   )
-  val legacyTalkRoute = Route[TalkPage, String](
```

Comment:
I don't know why this is here 😅
Maybe a remains of when I added the "history"

Reply:
C'était pour assurer la rétro-compat des liens en `scala.io/talks/*` (avant que tu n'ajoutes le segment `/<conference>/`) je pense.

Reply:
Ah oui c'est vrai, j'avais oublié 😅